packages/libs/user-datasets/src/lib/Components/UploadForm.scss

```diff
@@ -38,7 +38,6 @@
     display: flex;
     flex-direction: column;
     gap: 0.5em;
-    padding-left: 1em;
     padding-right: 1em;
```

Comment:
Maybe remove this padding-right, so the textarea boxes' right edges line up with the input group borders?